human_aware_rl/human/process_dataframes.py

```diff
@@ -17,7 +17,8 @@
 def get_human_human_trajectories(layouts, dataset_type, processed=False):
     """Get human-human trajectories"""
     assert dataset_type in ["train", "test"]
-    from human_aware_rl.imitation.behavioural_cloning import DEFAULT_BC_PARAMS
+    # please double check this
+    from human_aware_rl.imitation.behavior_cloning_tf2 import DEFAULT_BC_PARAMS
```

Comment:
@nathan-miller23  Please confirm if this is the desired behavior

Reply:
Overhauled this to work with the updated signatures